picker/style.css

```diff
@@ -41,6 +41,13 @@ main {
 	margin: 1em auto;
 	background: #f0f0f0;
 	box-shadow: .05em .05em .1em rgba(0,0,0,.2), 0 0 0 100vmax var(--color);
+	opacity: 0;
+
+	&:has(color-picker:defined) {
```

Comment:
Why fade in the entirety of `<main>`? Why not just the color-picker?

Reply:
Please take a look at how it might look like if we fade in the picker only:

https://github.com/user-attachments/assets/48b36a90-413d-46ea-b77d-b93de4b3384d

I like it better when it fades in as a whole unit: there is no layout shift and no empty text fields.


https://github.com/user-attachments/assets/5aed971b-3438-440b-813f-80cdcefb935a

But if you think fading in `<color-picker>` only is a better approach, we can do that. It's not a big change. :)